### PR TITLE
j/k movement for normal mode

### PR DESCRIPTION
**Pull request by Kazark**
Move up or down a line in normal mode. Handles boundary cases and scrolls as necessary. Left and right `h`/`l` movement not yet fully supported, though sketched out a little bit. Movements like `10j` are not yet supported by the normal mode bindings capabilities but are available in the backend already.

Also upgrade FsUnit version.

### CODE REVIEW

**Comment by Kazark** [comment]
Forgot to mention, this is another step for #19.

File: src/Void.ViewModel/RenderWindows.fs [hunk 1]
```diff
@@ -26,33 +26,83 @@ module RenderWindows =
...
-            [linesWithNoTilde..dimensions.Rows-1]
+            [linesWithNoTilde..(dimensions.Rows / 1<mRow> - 1)]
             |> List.map lineNotInBufferAsDrawingObject
 
         List.append (background :: bufferLines) rowsNotInBuffer
 
-    let private renderWindow (window : WindowView) =
-        let drawings = contentsAsDrawingObjects window.Dimensions window.Buffer
+    let private getCharacter (buffer : string list) cell =
+        buffer.[cell.Row/1<mRow>].[cell.Column/1<mColumn>].ToString()
```
Comment (mjgpy3) [inline src/Void.ViewModel/RenderWindows.fs]: I got an `IndexOutOfRangeException` here. I believe it happened when I scrolled over an empty line :)

Comment (Kazark) [inline src/Void.ViewModel/RenderWindows.fs]: Thanks for the find! Apparently the file I had been testing with had at least some whitespace in all its "blank" lines.

**Comment by Kazark** [comment]
Thanks for the fix! I merged it into my branch. If this looks good to you, feel free to merge.

**Pull request merged**